Documentation/RELEASE_NOTES.md

```diff
@@ -11,6 +11,7 @@
 ### Feature Enhancements
 * Add Geo-Redundant Control Node (GRCN) on SD-WAN Portal (METROAE-555)
 * Added ability to list installed plugins (METROAE-366)
+* All paths to files on the MetroAE host are relative with the exception of the path to image directories that need to be unzipped prior to MetroAE execution (METROAE-610)
```

Comment:
Let's keep this short. We can add more detail in CHANGES_IN_V5.md instead. I think it would be better to say something like `Add support for container-relative paths. More information can be found in <CHANGES_IN_V5.md>`. You can link the CHANGES_IN_V5.md file in the release note for easier navigation.